FR/2DCode/raw/recovery_ok.json

```diff
@@ -21,15 +21,15 @@
     }
```

Comment:
Value of ci should be dgci:V1:FR:EEVRA31SARJ3X5H0IT3V1IFES:31

Reply:
fix ok

Reply:
Verified OK